app/src/main/java/com/x8bit/bitwarden/ui/platform/components/image/BitwardenGifImage.kt

```diff
@@ -0,0 +1,51 @@
+package com.x8bit.bitwarden.ui.platform.components.image
+
+import android.content.res.Configuration
+import androidx.annotation.DrawableRes
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalConfiguration
+import com.bumptech.glide.integration.compose.ExperimentalGlideComposeApi
+import com.bumptech.glide.integration.compose.GlideImage
+
+/**
+ * A composable that displays a gif image.
+ *
+ * The content will also be scaled to fit the image to the container.
+ *
+ * **NOTE:** Due to using Glide under the hood, this does not respect the themed resource management
+ * of the passed in [resId] automatically. If there is a specific version of the `.gif` you can pass
+ * in the optional [darkModeResId]. You would need to save these in your resources with unique
+ * names.
+ *
+ * @sample
```

Comment:
The `@sample` actually messes up the docs, I think it is designed for a link. It should work if you just remove that one line.

Reply:
`@sample` can also reference a function.

![the_more_you_know](https://github.com/user-attachments/assets/13e09ad7-c8a6-45ff-9754-46e2a12a7705)